NitroxTest/Serialization/WorldPersistenceTest.cs

```diff
@@ -0,0 +1,444 @@
+﻿using System;
```

Comment:
Artifact from rebase probably. Should be removed.

Reply:
looks to be used below by Array reference.